programs/voter-stake-registry/tests/test_grants.rs

```diff
@@ -127,8 +127,10 @@ async fn test_grants() -> Result<(), TransportError> {
         .await;
     let deposit = &voter_data.deposits[1];
     assert_eq!(deposit.is_used, true);
-    assert_eq!(deposit.amount_deposited_native, 12000);
-    assert_eq!(deposit.amount_initially_locked_native, 12000);
+    let amount_deposited_native = deposit.amount_deposited_native;
```

Comment:
looks like unnecessary diff

Reply:
we have to have this due to `zero_copy(unsafe)` otherwise pointers would give undefined behavior. we need to copy the field to a local variable. 

![image](https://github.com/user-attachments/assets/2da9ffa3-bdce-42f6-aad3-861b990ed283)